### DISCUSSION TITLE
Sidebar collapse to menu button which does not respond

### DISCUSSION THREAD

**Discussion by pplanger** [Appsilon/shiny.fluent#175] [Q&A]
### Steps to reproduce

1. Open a shiny.fluent dashboard with a sidebar.
2. Decrease the size of your screen, e.g. typical mobile phone sizes

Below you can find a part of our code: 

`navigation <- Nav(
  
  IconButton.shinyInput("menu"),
  
  groups = list(
    list(links = list(
      list(name = 'Start', url = '#!/', key = 'home'),
      list(name = 'Browse Topics', url = '#!/browse-topics', key = 'browse'),
      list(name = 'Popular by Year', url = '#!/popular', key = 'popular'),
      list(name = 'Hot/Cold', url = '#!/hot-cold', key = 'hot-cold'),
      list(name = 'Topic Evolution', url = '#!/topic-evolution', key = 'topic-evolution'),
      list(name = 'Methods', url = '#!/methods', key = 'methods')
    ))
  ),
  initialSelectedKey = 'home',
  styles = list(
    linkText = list(
      fontWeight = 600,
      fontSize = 15,
      color = "white",
      ":hover" = list(
        color = "black"
      )
    ),
    root = list(
      height = '100%',
      width = "100%",
      boxSizing = 'border-box',
      overflowY = 'auto',
      overflowX = "hidden"
      # transition = 'width 0.3s ease-in-out',
      # selectors = list(
      #   ":hover" = list(
      #     width = "100%"
      #   )
      # )
    )
  )
)

menu = IconButton.shinyInput(
  "menu",
  iconProps = list(
    iconName = "CollapseMenu",
    styles = list(
      root = list(
        color = "#241b3e",
        fontWeight = 600
      )
    )
  )
)`

### Bug description

I'm currently trying to make our R-Shiny App more mobile-phone friendly. One of the main problems is that the sidebar collapses on mobile phones but the resulting button does not respond. As far as I have looked it up, this might be a bug of the shiny.fluent package. Does anyone of you know a simple solution to this issue?

### Expected behavior

The size of the sidebar should just decrease instead of collapsing or alternatively the resulting menu button should respond.  

### Comments

shiny.fluent app with sidebar: 
![Psychtopics with sidebar](https://user-images.githubusercontent.com/126692514/226869707-f0a1ed14-c60a-4271-b15f-9d7569759f66.PNG)

shiny.fluent app without sidebar (with not responding button):
![Psychtopics without sidebar](https://user-images.githubusercontent.com/126692514/226869781-06fc2e5d-a78c-4cd0-9791-c4eef31e715f.png)

**Comment by jakubsob** [comment]
Hi @pplanger,

This looks rather like an issue with CSS/JavaScript code used to collapse the sidebar, not a bug in shiny.fluent. The package provides only components and this is an issue with layout. 

The code you attached doesn't reproduce the issue. Please take a look what a [reproducible example](https://community.rstudio.com/t/faq-whats-a-reproducible-example-reprex-and-how-do-i-create-one/5219) and provide one. Otherwise we won't be able to determine if this is an bug in shiny.fluent or not.

**Comment by pplanger** [comment]
Hi @jakubsob,

thank you very much for your answer. 
You're absolutely right, it does not seem to be a bug of shiny.fluent. The issue is related to CSS/JavaScript. Sorry, for the confusion.
